FidelLimited/fidel-android
# Ux 585 add example applications

Added 4 example applications:

Java project which integrates the SDK with Jitpack
Kotlin project which integrates the SDK with Jitpack
Java project which embeds the SDK **without** Jitpack (but by importing a new module)
Kotlin project which embeds the SDK **without** Jitpack

**How to test**
Please run every application from the Example folder and check if the SDK is fine and if the `MainActivity` class is easy to understand :) That's where the example code is added.

Note: This branch, if approved, will be merged into `master` because these example applications are basically a form of documentation.
